eslint.config.js

```diff
@@ -0,0 +1,110 @@
+"use strict";
+
+const js = require("@eslint/js");
+const { FlatCompat } = require("@eslint/eslintrc");
+const eslintConfigESLintCJS = require("eslint-config-eslint/cjs");
+const globals = require("globals");
+const reactPlugin = require("eslint-plugin-react");
+const reactRecommended = require("eslint-plugin-react/configs/recommended");
+const jsxA11yPlugin = require("eslint-plugin-jsx-a11y");
+const reactHooksPlugin = require("eslint-plugin-react-hooks");
+
+const compat = new FlatCompat({
+    baseDirectory: __dirname
+});
+
+const [jsxA11yRecommended] = compat.extends("plugin:jsx-a11y/recommended");
+
+module.exports = [
+    {
+        ignores: [
+            "node_modules/**",
+            "_site/**",
+            "src/assets/js/**",
+            "src/_data/**",
+            "src/_11ty/**",
+            ".stylelintrc.js"
+        ]
+    },
+    ...eslintConfigESLintCJS,
+    {
+        files: ["**/*.js"],
+        languageOptions: {
+            globals: {
+                ...globals.node
+            }
+        },
+        rules: {
+            ...js.configs.recommended.rules,
+
+            // Disable eslint-plugin-node rules from eslint-config-eslint
+            "no-process-exit": "off",
+            "func-style": "off",
+            "n/no-deprecated-api": "off",
+            "n/no-extraneous-require": "off",
+            "n/no-missing-require": "off",
+            "n/no-unpublished-bin": "off",
+            "n/no-unpublished-require": "off",
+            "n/no-unsupported-features/es-builtins": "off",
+            "n/no-unsupported-features/es-syntax": "off",
+            "n/no-unsupported-features/node-builtins": "off",
+            "n/process-exit-as-throw": "off",
+            "n/shebang": "off",
+            "n/no-extraneous-import": "off",
+            "n/no-missing-import": "off",
+            "n/no-unpublished-import": "off",
```

Comment:
I would like to export a base config in `eslint-config-eslint`, so we don't need to disable these rules.

Reply:
That would be great.

Reply:
I agree that would be best. However, we can't release `eslint-config-eslint` at the moment because it enables some rules that are available only in v9 prereleases. What do you think about disabling `eslint-plugin-n` rules for now (as in this PR) until we release ESLint v9 final? Then, we'll release a new version of `eslint-config-eslint` that exports the base config and update this config to use that export.